docs/en/query_language/functions/type_conversion_functions.md

```diff
@@ -392,18 +392,77 @@ SELECT
 └───────────────────────────┴──────────────────────────────┘
 ```
 
-## parseDateTimeBestEffort {#type_conversion_functions-parsedatetimebesteffort}
+## parseDateTimeBestEffort {#parsedatetimebesteffort}
 
-Parse a number type argument to a Date or DateTime type.
-different from toDate and toDateTime, parseDateTimeBestEffort can progress more complex date format.
-For more information, see the link: [Complex Date Format](https://xkcd.com/1179/)
+Converts a string representation of a date and time value to its DateTime equivalent.
+Different from toDate and toDateTime, parseDateTimeBestEffort can progress more complex date format.
```

Comment:
I think we don't need this sentence. Refer these functions from **See Also**. Refer parseDateTimeBestEffort from the functions with the words about a wider range of input formats available.

Also, we always emphasize keywords with backticks (`toDate`) or with link([toDate](#)